community/releases/next-release-preview/index.md

```diff
@@ -55,3 +55,20 @@ Spinnaker's UI has changed! An application's nested menus are now represented as
 This change should not introduce any interruptions to a vanilla install of `deck`. However, if you've already made navigational changes to your group's instance of `deck` or created custom banners/headers for your app, you may need to make updates. The pattern for creating new routes in the side nav can be observed in the feature's PR:
 
 https://github.com/spinnaker/deck/pull/8239
+
+
+### Clouddriver SQL Cache Data Too Long Issue Resolved
+
+We've fixed an issue preventing data from being stored in Clouddriver's SQL 
+Cache because the data was too long 
+([Github issue](https://github.com/spinnaker/spinnaker/issues/5600)). As a part
+of this fix we introduced a second version of the tables used by Clouddriver
+for caching. You should see tables named `cats_v2_*` in your Clouddriver 
+database moving forward.
+
+Once you're comfortable with the Spinnaker release and don't expect to rollback 
+to a previous version then you can feel free to delete the first version of the 
+tables used for caching. In order to easily facilitate deleting these tables we
+have exposed an admin endpoint that will handle the deletion process. The admin
+endpoint can be reached via a curl PUT request against 
+`{your_clouddriver}/admin/db/drop_version/V1`.
```

Comment:
did we determine that only admins can hit these endpoints? might be worth highlighting!

Reply:
Yep, when validating the permission it explicitly checks if the user is an admin.